src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/IaasVm/PolicyTests.ps1

```diff
@@ -343,4 +343,54 @@ function Test-AzureVMPolicy
 		# Cleanup
 		Cleanup-ResourceGroup $resourceGroupName
 	}
+}
+
+function Test-AzureVMEnhancedPolicyAsDefault
+{ 
+	try
+    {
+		$resourceGroupName = "sgholapCZRTesting"
+		$vaultName = "sgholapZRSTestingVault"
+		$owner = "sgholap"
+		$AzureVMPolicyName = "AzureVMPolicy"
+		$AzureFilesPolicyName = "AzureFilesPolicy"
```

Comment:
add subscriptionId as a comment or variable to indicate the subscription used for this test